as2/server/redemption.md

```diff
@@ -0,0 +1,15 @@
+# Redemption
```

Comment:
Will need a secondary confirmation on this as I am not that familiar with the redemption protocol. Just as well, "unknown & not needed" needs to be 100% accurately confirmed before I can accept this. I have only encountered one instance in the protocol where this was true so far.